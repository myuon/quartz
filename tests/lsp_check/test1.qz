fun main(): i32 {
    return "foobar";
}
