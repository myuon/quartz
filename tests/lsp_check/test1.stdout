[{"end":[1,19],"message":"unification failed: string != i32","start":[1,11]}]
